src/internal.c

```diff
@@ -7047,6 +7068,11 @@ int InitSSL(WOLFSSL* ssl, WOLFSSL_CTX* ctx, int writeDup)
             return ret;
     }
 #endif
+
+#if defined(WOLFSSL_MAXQ10XX_TLS)
+    wolfSSL_maxq10xx_load_certificate(ssl);
```

Comment:
Check return value.

Reply:
Fixed in e529b95464797fba459ff2f0dbbad1c63b0d7444